File: src/PreventionAdvisor/wwwroot/app/workplaces/checklistItem.ts
import { Category } from './category';

export class ChecklistItem {

    constructor(workplaceId: string = null) {
        this.workplaceId = workplaceId;
    }

    id: string = '00000000-0000-0000-0000-000000000000';
    title: string = '';
    description: string = '';
    status: number = 2;
    categoryId: string = '';
    category: Category = new Category();
    workplaceId: string = '00000000-0000-0000-0000-000000000000';
}

File: src/PreventionAdvisor/Models/ChecklistItem.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace PreventionAdvisor.Models
{
    public class ChecklistItem
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Key]
        public Guid Id { get; set; }
        public String Title { get; set; }
        public String Description { get; set; }
        public int Status { get; set; }
    }
}
